src/pages/Restorans/Impressions/Impressions.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
This impressions component contains some UI bugs
![Screenshot 2023-01-23 at 10 38 07](https://user-images.githubusercontent.com/85338076/214009166-9a60c8e9-036a-4aa0-8877-2d271e2a0c8a.png)